index/views.py

```diff
@@ -58,15 +61,20 @@ def group_list(request):
     return render(request, 'index/group_list.html',{},
                 context_instance = RequestContext(request, processors = [custom_proc]))
 
-def custom_proc(request):    
-    vol = 0    
-    vol = random.randint(1,10)
+def custom_proc(request):
+    volumes = []
+    problems = Problem.objects.all().order_by('id')
+    ids = map(lambda problem: problem.id, problems)
```

Comment:
You can use https://docs.djangoproject.com/en/1.7/ref/models/querysets/#count instead of counting by yourself.